src/frontpage/components/Articles/MainArticlePlaceholder.tsx

```diff
@@ -0,0 +1,18 @@
+import classnames from 'classnames';
+import React from 'react';
+import { DefaultEventImage } from '../../../events/components/DefaultEventImage';
```

Comment:
https://github.com/dotkom/onlineweb-frontend/wiki/(Opinionated)-Structuring-and-ordering-of-imports